src/app/pages/user/user.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface IUser {
+  id: number;
+  userName: string;
+  userPassword: string;
```

Comment:
Would be better to mark userPassword as optional. Usually, we don't work with passwords on the front-end. So in most cases this field won't be exist